clients/ts/signalr-protocol-msgpack/README.md

```diff
@@ -8,7 +8,7 @@ npm install @aspnet/signalr-protocol-msgpack
 
 ## Usage
 
-See the [SignalR Documentation](https://docs.microsoft.com/en-us/aspnet/core/signalr) at docs.microsoft.com for documentation on the latest release.
+See the [SignalR Documentation](https://docs.microsoft.com/en-us/aspnet/core/signalr) and [JavaScript Documentation](https://docs.microsoft.com/javascript/api/%40aspnet/signalr-protocol-msgpack/?view=signalr-js-latest) at docs.microsoft.com for documentation on the latest release.
```

Comment:
This makes it sound like it's documentation on the JavaScript language/runtime :).

How about:

```
See the [SignalR Documentation](https://docs.microsoft.com/en-us/aspnet/core/signalr) at docs.microsoft.com for documentation on the latest release. [API Reference Documentation](https://docs.microsoft.com/javascript/api/%40aspnet/signalr-protocol-msgpack/?view=signalr-js-latest) is also available on docs.microsoft.com.
```